### ISSUE TITLE
Encrypt password before writing it to disk

### ISSUE THREAD

**Issue by btzy** [indocomsoft/fluminurs#51]
Here's a suggestion:

On Windows and OSX there are "proper" ways to encrypt passwords that are written on the disk, which do mitigate the security issue with writing unencrypted passwords to the disk.

For example, Windows has [`CryptProtectData`](https://docs.microsoft.com/en-us/windows/win32/api/dpapi/nf-dpapi-cryptprotectdata) which has the equivalent rust binding available.  Not sure about Linux, but it's a nice feature to have, even if it only works on Windows and OSX.

**Comment by indocomsoft** [comment]
Probably one can use https://github.com/hwchen/keyring-rs for a multi-platform solution

I won't have time to do this anytime soon, but PR's are always welcome

**Comment by btzy** [comment]
I see.  I probably won't be doing this anytime soon, because this feature isn't really a necessity.

**Comment by indocomsoft** [comment]
Yeah, no worries. Personally, the one I use daily is the Elixir one, so development is more active there. But it doesn't support non-Unix-y environment sadly
